### DISCUSSION TITLE
Equation Numbers is disturbed (katex).

### DISCUSSION THREAD

**Discussion by uga-rosa** [marp-team/marp#130] [Q&A] [answered]
Marp is very useful tool. However, there is one fatal defect for me, as I use a lot of mathematical formulas.
Equation number is not displayed correctly as shown below.
```
$$
a = 1 \tag{1}
$$
```
![スクリーンショット 2021-06-15 013201](https://user-images.githubusercontent.com/82267684/121927001-86b9ea00-cd79-11eb-9c8e-92f4cc5dfd83.png)
Can I solve this problem with CSS? I have confirmed this phenomenon with marp-cli and Marp for VS Code.

**Comment by yhatt** [comment, marked as answer]
Duplicated. See #124.

**Reply by uga-rosa** [reply]
Thx.
How to switch math renderer from KaTeX to MathJax in server mode of marp-cli?

**Reply by yhatt** [reply]
I wrote down in the unified docs: https://marp.app/docs/guide/math-typesetting#marp-cli.

[It is not ready to public](https://github.com/marp-team/marp/discussions/126) but some sections including math are already valuable.

See also: https://github.com/marp-team/marp-cli/issues/268#issuecomment-665207588

**Reply by uga-rosa** [reply]
I saw it but couldn't make it work, However, I just noticed a typo :_(
Thx!
